Fix test_analysisd/test_mitre/test_mitre_check_alert.py tests

The core team informs us that the `test_analysisd/test_mitre/test_mitre_check_alert.py` is not passing in the PR testing.

Indeed, I have checked and the following ones fail:

```
=================================== FAILURES ===================================
_ test_mitre_check_alert[/tmp/Test_integration_B4474_20210527190825/tests/integration/test_analysisd/test_mitre/data/test5.xml] _
test_analysisd/test_mitre/test_mitre_check_alert.py:52: in test_mitre_check_alert
    validate_mitre_event(event)
E   Failed: DID NOT RAISE <class 'jsonschema.exceptions.ValidationError'>
---------------------------- Captured stdout setup -----------------------------
Stopping wazuh-manager (via systemctl):  [  OK  ]
Starting wazuh-manager (via systemctl):  [  OK  ]
_ test_mitre_check_alert[/tmp/Test_integration_B4474_20210527190825/tests/integration/test_analysisd/test_mitre/data/test6.xml] _
test_analysisd/test_mitre/test_mitre_check_alert.py:52: in test_mitre_check_alert
    validate_mitre_event(event)
E   Failed: DID NOT RAISE <class 'jsonschema.exceptions.ValidationError'>
---------------------------- Captured stdout setup -----------------------------
Stopping wazuh-manager (via systemctl):  [  OK  ]
Starting wazuh-manager (via systemctl):  [  OK  ]
_ test_mitre_check_alert[/tmp/Test_integration_B4474_20210527190825/tests/integration/test_analysisd/test_mitre/data/test7.xml] _
test_analysisd/test_mitre/test_mitre_check_alert.py:52: in test_mitre_check_alert
    validate_mitre_event(event)
E   Failed: DID NOT RAISE <class 'jsonschema.exceptions.ValidationError'>
---------------------------- Captured stdout setup -----------------------------
Stopping wazuh-manager (via systemctl):  [  OK  ]
Starting wazuh-manager (via systemctl):  [  OK  ]
```

In these cases, invalid configurations are tested, where the `jsonschema.exceptions.ValidationError` exception is expected to be generated. It is possible that in these cases some **other exception may be generated**.

It is requested to review and fix these tests.

Regards.

--- a/deps/wazuh_testing/wazuh_testing/data/mitre_event.json
+++ b/deps/wazuh_testing/wazuh_testing/data/mitre_event.json
@@ -73,7 +73,9 @@
           "type": "object",
           "title": "The Mitre Schema",
           "required": [
-            "id"
+            "id",
+            "tactic",
+            "technique"
           ],
           "properties": {
             "id": {

--- a/docs/tests/integration/test_analysisd/test_mitre/test_mitre_check_alert.md
+++ b/docs/tests/integration/test_analysisd/test_mitre/test_mitre_check_alert.md
@@ -1,3 +1,30 @@
+# Test MITRE check alert
+
+## Overview 
+
+Check if `wazuh-analysisd` generates alerts enriching its fields with MITRE information.   
+
+## Objective
+
+The objective consists on checking if `wazuh-analysisd` can generate alerts using custom rules 
+that contains the `mitre` field to enrich those alerts with MITREs IDs, techniques and tactics.
+
+This test will check if the alert is syntactically and semantically correct. Otherwise, the 
+test will raise an exception.  
+
+## General info
+
+|Tier | Number of tests | Time spent |
+|:--:|:--:|:--:|
+| 0 | 15 | 42.4s |
+
+## Expected behavior
+
+- Fail if `wazuh-analysisd` doesn't generate the expected alert.
+- Fail if `wazuh-analysisd` generates an alert with syntactic or semantic errors.
+- Fail if `wazuh-analysisd` generates a correctly formatted alert for those 
+  cases with invalid configurations.
+
 ## Code documentation
 
 ::: tests.integration.test_analysisd.test_mitre.test_mitre_check_alert

--- a/docs/tests/integration/test_authd/test_remote_enrollment.md
+++ b/docs/tests/integration/test_authd/test_remote_enrollment.md
@@ -32,5 +32,4 @@ It is executed for stand alone manager and also for cluster manager and worker[2
 
 ## Code documentation
 
-
-::: tests.integration.test_authd.test_remote_enrollment -->
+::: tests.integration.test_authd.test_remote_enrollment

--- a/tests/integration/test_analysisd/test_mitre/test_mitre_check_alert.py
+++ b/tests/integration/test_analysisd/test_mitre/test_mitre_check_alert.py
@@ -25,8 +25,8 @@ configurations = []
 for i in range(1, 15):
     file_test = os.path.join(_data_path, f"test{i}.xml")
     configurations.append(file_test)
-    if i in range(5,9):
-        invalid_configurations.append(os.path.join(_data_path, f"test{i}.xml"))
+    if i in range(5, 9):
+        invalid_configurations.append(file_test)
 
 
 # fixtures

